### PR TITLE
Adding in several new scripts to gather system info

### PR DESCRIPTION
**Pull request by Belgabad3435**

### CODE REVIEW

File: tar/tar_logs.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Copyright (c) 2017 All Rights Reserved, http://www.threatstack.com
+echo "/* Tarring log files ***************************************************/"
+current_date="$(date +'%m-%d-%Y')"
+tar -cvzf ts_logs$current_date.tar.gz /opt/threatstack/cloudsight/logs/ || true
```
Comment (yankeehacker) [inline tar/tar_logs.sh]: Maybe have another `_` between `logs` and `$current_date`, so it says `ts_logs_$current_date`

Comment (yankeehacker) [inline tar/tar_logs.sh]: Do we want to capture any of the system logs here, or is that a separate script?

File: diagnostics.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Copyright (c) 2017 All Rights Reserved, http://www.threatstack.com
+echo "/* BEGINNING THREAT STACK DIAGNOSTICS **********************************/"
+sudo bash system/gather_system_info.sh | sudo tee /opt/threatstack/cloudsight/logs/system_info.txt
+sudo bash docker/gather_docker_info.sh | sudo tee /opt/threatstack/cloudsight/logs/docker_info.txt
```
Comment (yankeehacker) [inline diagnostics.sh]: I'm not sure if we want to make this an option, but how do we envision using this? Do we envision our customers checking out the repo, individual scripts, or should we package and ship out a suite of scripts? This script assumes a certain directory layout.

Comment (Belgabad3435) [inline diagnostics.sh]: The directory layout can be easily changed. My vision for this is eventually having this packaged with the agent, so we can have the customer run a command that's already on their system, which generates the data that we need.

File: tar/tar_logs.sh (hunk 1 above)
Comment (Belgabad3435) [inline tar/tar_logs.sh]: I added the extra underscore. I agree that that's a little more clear. I think capturing necessary system logs here makes sense (as opposed to another script), but I'm not sure if that's something that needs to be added in now

**Pull request merged**